components/StyledButton.js

```diff
@@ -1,30 +1,22 @@
-import styled, { css } from "styled-components";
+import styled from "styled-components";
 
 export const StyledButton = styled.button`
   width: 8rem;
   height: 8rem;
   border-radius: 50%;
   background-color: #546391;
   padding: 0.8rem;
- 
   color: #EBE1CA;
   text-decoration: none;
   font-weight: bold;
   border: 0px;
-  font-size: inherit;
+  font-size: 1.2rem;
   box-shadow: 0 0px 25px 2px #EBE1CA, 0 0px 5px 0px #ffffff, inset 0 0px 25px 2px #EBE1CA;
```

Comment:
you can remove those units

```suggestion
  box-shadow: 0 0 25px 2px #EBE1CA, 0 0 5px 0 #ffffff, inset 0 0 25px 2px #EBE1CA;
```